go/vt/vtgate/gateway/discoverygateway_test.go

```diff
@@ -32,6 +32,10 @@ import (
 	vtrpcpb "github.com/youtube/vitess/go/vt/proto/vtrpc"
 )
 
+func tabletIdent(tablet *topodatapb.Tablet) string {
+	return fmt.Sprintf("%s-%d (%s)", tablet.Alias.Cell, tablet.Alias.Uid, tablet.Hostname)
+}
+
```

Comment:
Let's move that function to go/vt/topotools/tablet.go, export it and name it TabletIdent.

Additionally, it would be nice to also have in there the 'tags' fields, something like:
cell-123 (host1 tag1=value1)
The tags are usually used to describe a tablet (and for instance put some extra name in there, that's what we do inside YouTube). They are displayed in 'vtctl ListAllTablets' and it's very useful.